### PR TITLE
feat: share post

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
- Added share post option and fix suggested user follow.

### CODE REVIEW

File: src/common/components/SuggestionCard.jsx [hunk 1]
```diff
@@ -8,38 +10,52 @@ export const SuggestionCard = ({
...
-          src={profileImage.url}
-          alt={profileImage.original_filename}
-          className="w-10 h-10 mr-4 object-cover rounded-full flex-shrink-0 bg-gray-200"
-        />
-      ) : (
-        <div className="w-10 h-10 mr-4 text-lg flex items-center flex-shrink-0 justify-center font-semibold rounded-full bg-blue-500 text-white">
-          {firstName[0].toUpperCase()}
+    <div className="flex items-center p-2 pt-3 mb-4 rounded-lg shadow border dark:bg-gray-800 dark:border-gray-700">
+      <Link
+        title={`@${username}`}
```
Comment (ishitakeshawani) [inline src/common/components/SuggestionCard.jsx:22]: What is use of this title here?

**Review by ishitakeshawani — APPROVED** [review]
Code is clean
UI is nice
good to merge

File: src/common/components/SuggestionCard.jsx (hunk 1 above)
Comment (rohanmathur91) [reply src/common/components/SuggestionCard.jsx:22]: it provides more information about the tag, like a tooltip

**Review by anupkgurung — APPROVED** [review]
clean code.
Good UI and UX.

**Pull request merged**